src/Components/MovieDetails/MovieDetails.js

```diff
@@ -11,20 +11,39 @@ class MovieDetails extends Component {
   }
   componentDidMount = () => {
     fetchSingleMovie(`movies/${this.props.movieID}`)
-      .then((data) => console.log(data))
+      .then((data) => this.setState({ singleMovie: data.movie }))
       .catch((error) => console.log("error", error));
```

Comment:
Rather than just console logging the error - maybe we could do something with it, like displaying it to the user?